e2etest/consts/const.go

```diff
@@ -0,0 +1,12 @@
+package consts
+
+const (
+	// BGPTestFRRConfigDir contains the name of the BGP configuration directory
+	BGPTestFRRConfigDir = "config/frr"
```

Comment:
Please move the constants used in a single package to that package. We need a shared package only if multiple packages access the same constant.